buzsaki_lab_to_nwb/valero/eventsinterface.py

```diff
@@ -0,0 +1,211 @@
+
+        mat_file = read_mat(up_down_states_file_path, variable_names=["UDStates"])
+
+        up_and_down_states_data = mat_file["UDStates"]
+
+        intervals = up_and_down_states_data["ints"]
+        up_intervals = intervals["UP"]
+        up_start_time, up_stop_time = up_intervals[:, 0], up_intervals[:, 1]
+        down_intervals = intervals["DOWN"]
+        down_start_time, down_stop_time = down_intervals[:, 0], down_intervals[:, 1]
+
+        # Combine and sort UP and DOWN intervals
+        combined_start_times = np.concatenate((up_start_time, down_start_time))
+        combined_stop_times = np.concatenate((up_stop_time, down_stop_time))
+        combined_states = np.array(["UP"] * len(up_start_time) + ["DOWN"] * len(down_start_time))
+
+        # Create an array of indices that sorts the start times
+        sort_indices = np.argsort(combined_start_times)
+
+        # Sort all arrays using the sorting indices
+        combined_start_times = combined_start_times[sort_indices]
+        combined_stop_times = combined_stop_times[sort_indices]
+        combined_states = combined_states[sort_indices]
+
+        # Create TimeIntervals
+        name = "UpDownStatesTimeIntervals"
+        description = "TBD"
+        states_intervals = TimeIntervals(name=name, description=description)
+
+        # Add a new column for states
+        states_intervals.add_column(name="state", description="State (UP or DOWN)")
+
+        # Add intervals
+        for start_time, stop_time, state in zip(combined_start_times, combined_stop_times, combined_states):
+            states_intervals.add_interval(start_time=start_time, stop_time=stop_time, state=state)
+
+        processing_module = get_module(nwbfile=nwbfile, name="behavior")
+        processing_module.add(states_intervals)
+
+
+class ValeroHSEventsInterface(BaseDataInterface):
+    def __init__(self, folder_path: FolderPathType):
+        super().__init__(folder_path=folder_path)
+
+    def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = False):
+        self.session_path = Path(self.source_data["folder_path"])
+        self.session_id = self.session_path.stem
+
+        # We use the behavioral cellinfo file to get the trial intervals
+        hse_data_path = self.session_path / f"{self.session_id}.HSE.mat"
+        if not hse_data_path.exists():
+            warnings.warn(f"HSE event file not found: {hse_data_path}. Skipping HSE events interface.")
+            return nwbfile
+
+        mat_file = read_mat(hse_data_path, variable_names=["HSE"])
+        hse_data = mat_file["HSE"]
+
+        hse_intervals = hse_data["timestamps"]
+        peaks = hse_data["peaks"]
+        center = hse_data["center"]
+
+        column_descriptions = dict(
+            peaks="TBD",
+            center="TBD",
+        )
+
+        column_data_dict = dict(
+            peaks=peaks,
+            center=center,
+        )
```

Comment:
These had the same issue as the main ripple interface; these are definitely time values and so should end in `_time` at least